packages/components/src/Autocomplete/Autocomplete.css

```diff
@@ -17,6 +17,14 @@
   background-color: var(--color-white);
 }
 
+.options.medium {
+  max-height: 300px;
```

Comment:
I really don't like this but unfortunately this is how we define our line item picker height in jobber online. This is a concession to maintain visual parity.

Reply:
I would add your comment as a CSS comment here. We'll wonder why later...